ios/RNRadar.m

```diff
@@ -321,6 +321,9 @@ - (void)didLogMessage:(NSString *)message {
 
 RCT_EXPORT_METHOD(startTrip:(NSDictionary *)optionsDict resolve:(RCTPromiseResolveBlock)resolve reject:(RCTPromiseRejectBlock)reject) {
     RadarTripOptions *options = [RadarTripOptions tripOptionsFromDictionary:optionsDict];
+    if (options.scheduledArrivalAt) {
+        options.scheduledArrivalAt = [RCTConvert NSDate:options.scheduledArrivalAt];
```

Comment:
What does this do?

Reply:
React Native Bridge can't deliver Date object itself, so we need to deliver Date value (Number) and on iOS we need to convert to NSDate value.

Reply:
Perfect. Got it.